content/en/docs/concepts/security/certificates.md

```diff
@@ -0,0 +1,578 @@
+* The X.509 Subject field Subject Alternate Names extension are copied verbatim
+  from `spec.request`.
+* The "Key Usage" and "Extended Key Usage" extensions are set according to the key
+  usages named in `spec.usages`.
+* No other extensions are included.
+* NotBefore is set to the present time, with reasonable backdating to
+  account for clock skew.
+* NotAfter is set to the present time plus the minimum of `spec.expirationSeconds`
+  and the certificate TTL configured for `kube-controller-manager` (365 days, by
+  default), with reasonable backdating to account for clock skew.
+* The CA bit to false.
+
+### `kubernetes.io/kubelet-serving` {#built-in-signer-kube-apiserver-client-kubelet-serving}
+
+This signer issues certificates that the API server will honor as valid
+`kubelet` serving certificates.  It offers no further guarantees on what the
+certificates may be used for.
+
+This signer responds only to CertificateSigningRequests, not
+PodCertificateRequests.
+
+There is no built-in auto-approval controller for this signer.
+
+This signer does not distribute its trust bundle, since no party besides the API
+server is intended to verify the certificates it issues.
+
+When this signer's signing controller sees an approved
+CertificateSigningRequest, it will check the following conditions:
+* The `Subject.CommonName` field has the prefix `system:node:`
+* There is one `Subject.Organization` entry, exactly equal to `system:nodes`
+* There is at least one DNS or IP entry in Subject Alternate Name extension.
+* There are no Email or URI entries in the Subject Alternate Name extension.
+* The CSR `spec.usages` field is either:
+  * `digital signature` and `server auth`, or
+  * `digital signature`, `server auth`, and `key encipherment`. 
+
+If the conditions are true, the signing controller will issue the certificate.
+Otherwise it will mark the CSR as failed.
+
+The issued certificate has the following properties:
+* The X.509 Subject field Subject Alternate Names extension are copied verbatim
+  from `spec.request`.
+* The "Key Usage" and "Extended Key Usage" extensions are set according to the key
+  usages named in `spec.usages`.
+* No other extensions are included.
+* NotBefore is set to the present time, with reasonable backdating to
+  account for clock skew.
+* NotAfter is set to the present time plus the minimum of `spec.expirationSeconds`
+  and the certificate TTL configured for `kube-controller-manager` (365 days, by
+  default), with reasonable backdating to account for clock skew.
+* The CA bit to false.
+
+### `kubernetes.io/kube-apiserver-serving` {#built-in-signer-kube-apiserver-serving}
+
+{{< feature-state feature_gate_name="ClusterTrustBundle" >}}
+
+This signer does not issue any certificates.  It purely serves to distribute the
+trust anchors that allow workloads within the cluster to verify the API server's
+server certificate when they connect to `kubernetes.default.svc`.
```

Comment:
`kubernetes.defaults.svc` is only one of the ways to connect to the kube-apiserver. See https://github.com/kubernetes/website/pull/48492/files#diff-6d12239777c4a9144a2fb9051645d7ace1f7d2a4a79f28e819c1dafb4bde0abeR219-R220